python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -45,35 +45,23 @@
 __all__ = ["DcrAssembleCoaddConnections", "DcrAssembleCoaddTask", "DcrAssembleCoaddConfig"]
 
 
-class DcrAssembleCoaddConnections(pipeBase.PipelineTaskConnections,
+class DcrAssembleCoaddConnections(AssembleCoaddConnections,
                                   dimensions=("tract", "patch", "band", "skymap"),
-                                  defaultTemplates={"inputCoaddName": "deep",
+                                  defaultTemplates={"inputWarpName": "deep",
```

Comment:
Is the inputWarpName template here in case you want to have different warps and coadds? Is that a possible or expected mode of operation?

Reply:
Yes, these will be different in general. I'll add a comment.